diagnostics-app/src/app/routine-v2/routine-v2-card/routine-v2-card-content/routine-v2-card-content.component.html

```diff
@@ -0,0 +1,18 @@
+<!--
+ Copyright 2023 The Chromium Authors. All rights reserved.
+ Use of this source code is governed by a BSD-style license that can be
+ found in the LICENSE file.
+-->
+<ng-container
+  *ngIf="isPrimitiveType(output); then renderPrimitive; else renderSubobject"
+></ng-container>
+<ng-template #renderPrimitive>
+  {{ output }}
+</ng-template>
+<ng-template #renderSubobject>
+  <div class="routine-v2-card-content" *ngFor="let item of $any(output) | keyvalue: originalOrder">
+    <b>{{ item.key }}</b
+    >:
```

Comment:
Why new line here?

Reply:
Seems to be a formatter issue with `:` after `>`. Done.